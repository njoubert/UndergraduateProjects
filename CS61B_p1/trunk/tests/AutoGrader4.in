Roman lives in the russian house.
Dmitri does not live in the ukranian house.
Wolodja does not live in the russian house.
There is a belorussian house.
Where does Dmitri live?
Where does Wolodja live?

