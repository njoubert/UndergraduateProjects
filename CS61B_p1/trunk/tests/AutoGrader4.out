1. Roman lives in the russian house.
2. Dmitri does not live in the ukranian house.
3. Wolodja does not live in the russian house.
4. There is a belorussian house.
Q: Where does Dmitri live?
A: Dmitri lives in the belorussian house.
Q: Where does Wolodja live?
A: Wolodja lives in the ukranian house.
